app/admin/user.rb

```diff
@@ -22,27 +26,31 @@
       f.input :first_name
       f.input :last_name
       f.input :email
-      f.input :barcode
       f.input :alt_email
-      f.input :alt_barcodes
+      if f.object.new_record?
```

Comment:
I'm not sure what alt_barcodes does.  It feels like maybe it's redundant and should be deleted from both user model and admin user model.  If so, let's bring it back here, and make a new PR for that task.

Reply:
@daryanypl Courtney was pleased to see alt_barcodes removed from the form.  There are 3000 users that have an alt_barcode values like this: `;277770XXXXXXXX;277770XXXXXXXX` or this: `277770XXXXXXXX` so maybe the best thing is just to leave the data in the database?

Reply:
If they're in the database and being used, we should provide a way to see and edit.  If they're in the database and being phased out, we should provide a way to see, but not edit (show the field, but gray it out and make it inactive with a little "deprecated" tag).  If we're getting rid of them, it should be its own separate pull request and user story.

The goal is to make sure that a pull request contains only the code that's relevant to that pull request.  None of us are perfect at it, but we try.

Reply:
@daryanypl you mentioned verbally you'd create another ticket for this.  Thank you.
This is ready for your final review.

Reply:
Yep, made https://jira.nypl.org/browse/MLN-308 .